AzurePSDrive.psm1

```diff
@@ -4,25 +4,25 @@ using module .\AzurePSDriveStorageAccount.psm1
 using module .\AzurePSDriveVM.psm1
 using module .\AzurePSDriveWebApp.psm1
 
-$script:AzureRM_Profile = if($IsCoreCLR){'AzureRM.Profile.NetCore'}else{'AzureRM.Profile'}
-$script:AzureRM_Resources = if($IsCoreCLR){'AzureRM.Resources.Netcore'}else{'AzureRM.Resources'}
-$script:pathPattern = [System.IO.Path]::Combine('Azure:', '*', 'ResourceGroups', '*')
+$script:Az_Profile = 'Az.Profile'
+$script:Az_Resources = 'Az.Resources'
+$script:pathPattern = [System.IO.Path]::Combine('Az:', '*', 'ResourceGroups', '*')
```

Comment:
Does this change the name of the root directory shown to the user?

Reply:
Actually this is the question I would like your guys today.  Should we take this chance to change the `Azure` drive to `Az` as well?  I made the change already. But if we don't think this is a good idea, I will back it out easily.